Dockerfile.tmpl

```diff
@@ -201,7 +201,7 @@ RUN pip install pysal \
 
 RUN apt-get install -y libfreetype6-dev && \
     apt-get install -y libglib2.0-0 libxext6 libsm6 libxrender1 libfontconfig1 --fix-missing && \
-    # b/198300835 kornia 4.1.0 is not compatible with our version of numpy.
+    # b/198300835 gensim 4.1.0 is not compatible with our version of numpy.
     pip install gensim==4.0.1 \
```

Comment:
Are we using a newer version of numpy now? If so, can we remove the `gensim` pin too?

Reply:
I'm working on a different PR for gensim: https://github.com/Kaggle/docker-python/pull/1233

Reply:
I checked and we are now using numpy 1.21. 

I tested with the latest version of gensim (4.2.0) and it works.

Safe to remove the pin.